### ISSUE TITLE
Integrating the dashlet library with sample data in cQube 4.0 alpha.

### ISSUE THREAD

**Issue by TLSravani** [Sunbird-cQube/community#143]

**Comment by pandutibil** [comment]
what is the Requirement?
Need to integrate dashlets library which supports a variety of charts, maps, and other components into cQube 4.0.alpha.

where the new development is required? (whether it is Ingestion or processing or configuration or reference visualization app)
It is a part of the reference visualization app.

when can finish the requirement?
Integration will be completed by 12-09-2022

**Comment by pandutibil** [comment]
12-09-2022 & 13-09-2022

While installing the dashlet into cQube 4.0, I faced dependency issues with the main project. After fixing all the dependency issues, then tried to integrate the dashlets. While integrating the dashlets faced some issues like leaflet and data table components were not rendered due to some JavaScript files are missing. Instead of including the missing JavaScript files(Jquery, leaflet, datatable.js) into cQube 4.0, searched for the solution to export all these dependency scripts in the production build of the library but didn't get any solution. After that I included the files in the cQube 4.0, then the map and datatable components were rendered.
